suite-common/wallet-core/src/device/deviceThunks.ts

```diff
@@ -345,6 +352,13 @@ export const authorizeDevice = createThunk(
     },
 );
 
+export const reAuthorizeDevice = createThunk(
```

Comment:
```suggestion
export const reAuthorizeDeviceThunk = createThunk(
```

Reply:
Is this thunk needed? what about calling `authorizeDevice` thunk directly from `suite-native retryPassphraseAuthentication`

Reply:
Good point, it's redundant. Fixed and rebased.